src/blas/impl/KokkosBlas1_nrm2_impl.hpp

```diff
@@ -118,112 +119,53 @@ struct V_Nrm2_Functor {
 };
 
 /// \brief Column-wise 2-norm functor for multivectors; works for
-///   any layout, but best performance with LayoutRight.
+///   any layout, but best performance with LayoutLeft.
```

Comment:
This comment is for the functor but it would be nice to print a warning in the `invoke` routine if the user provides a LayoutRight view. Going forward, consider extending with optimized paths for both LayoutRight and LayoutLeft.

Reply:
@e10harvey I do have in mind a way to tweak these functors to be coalesced with LayoutRight, I just didn't add them yet because a) the main user of these functions is Tpetra MultiVector which is always LayoutLeft, and b) even for LayoutRight views the new code is faster than the old code. The old code had each GPU thread iterate through rows sequentially, so there was some spatial locality (relying on cache) but it didn't coalesce.

Reply:
OK, consider filing an issue on your backlog for LayoutRight.